Source/WebCore/rendering/ios/RenderThemeIOS.h

```diff
@@ -59,11 +59,6 @@ class RenderThemeIOS final : public RenderThemeCocoa {
 
     WEBCORE_EXPORT static Color systemFocusRingColor();
 
-    struct IconAndSize {
-        RetainPtr<UIImage> icon;
-        FloatSize size;
-    };
-
     WEBCORE_EXPORT static IconAndSize iconForAttachment(const String& fileName, const String& attachmentType, const String& title);
```

Comment:
You *might* need to specify `RenderThemeCocoa::IconAndSize`, but if the compiler is happy I guess it's good!